packit/local_project.py

```diff
@@ -442,6 +444,14 @@ def create_branch(
 
         return head
 
+    def _fetch_as_branch(
+        self, remote_ref: str, local_ref: str, local_branch: str
+    ) -> None:
+        remote = self.remote or "origin"
```

Comment:
docstring please, ideally with equal git commands to make this easier to read and understand :)